### DISCUSSION TITLE
Slow throughput downloading files over USB (~11MB/s)

### DISCUSSION THREAD

**Discussion by rupello** [gopro/OpenGoPro#399] [General]
I'm seeing only around 11MB/s when downloading large files from HERO9 over USB.  I'm seeing the same thing on Mac & Windows. 

Transfers over MTP seem to be around 2-3x faster

I'm interested to know if others are seeing similar throughputs?

**Comment by kmatzen** [comment]
I'm getting 30 MB/s from Hero10 to macOS over USB.
